src/sentry/snuba/metrics.py

```diff
@@ -500,7 +500,7 @@ class SnubaField:
                 # Subset of possible states:
                 # https://develop.sentry.dev/sdk/event-payloads/transaction/
                 "ok",
-                "cancelled",
+                "canceled",
```

Comment:
```suggestion
                "cancelled",
```

Reply:
:D @jjbayer told me that this was probably a typo and I could fix it